js/interface.js

```diff
@@ -231,7 +229,6 @@ var DynamicLists = (function() {
           var value = $(this).val();
           var type = $(this).data('field');
           var $selector = $(this).parents('.filter-panel');
-          var id = $(this).attr('filter-item-id');
```

Comment:
This doesn't match the changes in https://github.com/Fliplet/fliplet-widget-dynamic-lists/pull/454/files and so it doesn't look like a correct revert.